### ISSUE TITLE
Cant unmap default keybinds.

### ISSUE THREAD

**Issue by chrismit3s** [phaazon/mind.nvim#65]
I dont see any way to avoid a default mapping to be created, which is problematic because the mind keymaps override my other keymaps.

**Comment by B4rc1** [comment]
Same here, tried setting the command to `nil` but even that did not work unfortunately.
Looking at [the code](https://github.com/phaazon/mind.nvim/blob/e59c52758c399caceb549c698cfa2d65e6bbb9f9/lua/mind/keymap.lua#L51-L89) this is not possible. So this is more of a feature request than a bug report.

**Comment by chrismit3s** [comment]
Yes, this actually made the plugin unusable for me. Eventually I switched to [Logseq](https://logseq.com/) for notetaking. Its awesome! Fully open source and a great android app.
